pages/installation/os/ubuntu.rst

```diff
@@ -39,7 +39,7 @@ The last step is to enable MongoDB during the operating system's startup::
 Elasticsearch
 -------------
 
-Graylog can be used with Elasticsearch 6.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/deb.html>`__::
+Graylog can be used with Elasticsearch 6.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/6.8/deb.html>`__::
```

Comment:
You can also use `https://www.elastic.co/guide/en/elasticsearch/reference/6.x/deb.html` afaics, so we do not have to change the url for 6.9ff.